src/Databases/MySQL/DatabaseMaterializedMySQL.cpp

```diff
@@ -137,7 +137,7 @@ StoragePtr DatabaseMaterializedMySQL::tryGetTable(const String & name, ContextPt
     if (context_->isInternalQuery())
         return nested_storage;
     if (nested_storage)
-        return std::make_shared<StorageMaterializedMySQL>(std::move(nested_storage), this);
+        return StorageMaterializedMySQL::create(std::move(nested_storage), this);
```

Comment:
this change is the opposite of intended

Reply:
Yepp, thanks, will fix.

With `shared_ptr_helper`/`create()` gone, this does in theory not even build. My preliminary dev setup has the strange issue that it doesn't build MySQL support by default, so I didn't notice this problem when building locally. I currently think it has to do with the CMake cache which stores MYSQL as an option, will try with a flushed CMakeCache.